src/views/waypoint/home/components/maybe-internal-link/index.tsx

```diff
@@ -0,0 +1,29 @@
+import Link from 'next/link'
+import isAbsoluteUrl from 'lib/is-absolute-url'
+import React from 'react'
+
+function MaybeInternalLink({
+  href,
+  children,
+  ...rest
+}: {
+  href: string
+  children: React.ReactNode
+} & React.HTMLProps<HTMLAnchorElement>): React.ReactElement {
+  const Elem = isAbsoluteUrl(href) ? InternalLink : 'a'
+  return (
+    <Elem href={href} {...rest}>
+      {children}
+    </Elem>
+  )
+}
```

Comment:
I could definitely see this being useful elsewhere! 💭

Reply:
Sweet - I've ended up lifting this up into `components` to get client-side links in the `sidebar` going 👍